crates/astria-composer/src/executor/mod.rs

```diff
@@ -689,7 +689,7 @@ impl Future for SubmitFut {
                     info!(
                         nonce.actual = *this.nonce,
                         bundle = %telemetry::display::json(&SizedBundleReport(this.bundle)),
-                        transaction.hash = %telemetry::display::hex(&tx.sha256_of_proto_encoding()),
+                        transaction.hash = tx.id().get(),
```

Comment:
This would give you a very ugly string of the form `[2, 5, 3441, 331, ...]`. Instead, use the display implementation that you defined for the ID:

```suggestion
                        transaction.id = %tx.id(),
```